src/test/java/unit/com/mathworks/ci/actions/RunMatlabTestsActionTest.java

```diff
@@ -170,11 +170,4 @@ public void printsAndRethrowsMessage() throws IOException, InterruptedException,
             assertEquals(12, e.getExitCode());
         };
     }
-
-    @Test
-    public void shouldRemoveTempFolder() throws IOException, InterruptedException, MatlabExecutionException {
-        action.run();
-
-        verify(runner).removeTempFolder();
-    }
```

Comment:
This test was removed, but I don't think that `RunMatlabTestsAction.java` was updated to use the new teardown method.

Reply:
Oh yeah, I think we can start using the `teardownActions()` method in `RunMatlabTestsAction` as well. Will update it, thanks!